packages/framework/src/Model/TransportAndPayment/FreeTransportAndPaymentFacade.php

```diff
@@ -40,12 +40,12 @@ public function isActive(int $domainId, bool $forceFreeTransportAndPayment): boo
     }
 
     /**
-     * @param \Shopsys\FrameworkBundle\Component\Money\Money $productsPriceWithVat
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\PriceInterface $productsPrice
      * @param int $domainId
      * @param bool $forceFreeTransportAndPayment
      * @return bool
      */
-    public function isFree(Money $productsPriceWithVat, int $domainId, bool $forceFreeTransportAndPayment): bool
+    public function isFree(PriceInterface $productsPrice, int $domainId, bool $forceFreeTransportAndPayment): bool
```

Comment:
this should be mentioned in the upgrade notes IMHO

Reply:
I thought we no longer mention changes in method interfaces as those are reported by coding standards.